internal/compiler/compat.go

```diff
@@ -16,10 +16,6 @@ func stringSlice(list *ast.List) []string {
 			items = append(items, n.Str)
 			continue
 		}
-		if n, ok := item.(*ast.String); ok {
-			items = append(items, n.Str)
-			continue
-		}
```

Comment:
Can you explain why this needs to be removed?

Reply:
If you look at the full function the condition is there twice.
I was tidying a little bit the code, it does not impact anything.
```
func stringSlice(list *ast.List) []string {
	items := []string{}
	for _, item := range list.Items {
		if n, ok := item.(*ast.String); ok {
			items = append(items, n.Str)
			continue
		}
		if n, ok := item.(*ast.String); ok {
			items = append(items, n.Str)
			continue
		}
```